libraries/AP_EFI/AP_EFI_State.h

```diff
@@ -199,4 +204,32 @@ struct EFI_State {
     // Status for each cylinder in the engine
     Cylinder_Status cylinder_status[ENGINE_MAX_CYLINDERS];
 
-};
+    //custom parameters CX
```

Comment:
This is all for logging, probably?

If you want to log it, make it private state in the Hirth backend.

Reply:
Moved to Hirth.cpp